src/picongpu/include/fields/FieldManipulator.hpp

```diff
@@ -61,7 +61,8 @@ class FieldManipulator
                 uint32_t pos_or_neg = i % 2;
 
                 uint32_t thickness = ABSORBER_CELLS[direction][pos_or_neg];
-                float_X absorber_strength = ABSORBER_STRENGTH[direction][pos_or_neg];
+
+                float_X absorber_strength = -1.0 * math::log( ABSORBER_MAXFACTOR[direction][pos_or_neg] ) * thickness;
```

Comment:
This should be `-1.0 * math::log( ABSORBER_MAXFACTOR[direction][pos_or_neg] ) / thickness;` ??